core/controllers/profile.py

```diff
@@ -77,6 +79,17 @@ def get(self, username):
         if not user_settings:
             raise self.PageNotFoundException
 
+        user_contributions = user_services.get_user_contributions(
+            user_settings.user_id)
+
+        created_exploration_summaries = (
+            exp_services.get_displayable_exploration_summaries_matching_ids(
+                user_contributions.created_exploration_ids))
+
+        edited_exploration_summaries = (
+            exp_services.get_displayable_exploration_summaries_matching_ids(
```

Comment:
If you're passing these to the frontend, you will have to convert them to dicts (and call them created_exploration_summary_dicts etc).